recipes/rnabridge-denovo/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "rnabridge-denovo" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/Shao-Group/rnabridge-denovo/releases/download/v{{ version }}/rnabridge-denovo-{{ version }}.tar.gz"
+  sha256: "1bfe3dd9fa2f9151aa7bbc36b7308a9fe9a1355415a4dc5dbd99b8eadf91c221"
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}

```

Comment:
the C compiler is not really needed, isn't it?

Reply:
> the C compiler is not really needed, isn't it?
No, it's not needed.